tests/dmctl_basic/conf/baz-task.yaml

```diff
@@ -0,0 +1,25 @@
+name: baz-task
+task-mode: all
+shard-mode: pessimistic
+
+target-database:
+  host: "127.0.0.1"
+  port: 4000
+  user: "root"
+  password: ""
+
+routes:
+  route-rule-1:
+    schema-pattern: "test"
+    table-pattern: "baz*"
+    target-schema: "test"
+    target-table: "baz"
```

Comment:
just for test whether pause/resume/stop works as expected, just a dummy task

Reply:
Ok, I mean your task just pause/resume/stop, don't execute sql to load and syncer, so that you don't use the `route` module and `black-white list` as follow.

If that, maybe you can remove it.

Reply:
all integration test uses same mysql instances, `black-white list` is used to isolate cases.

`route` can be removed though.